components/widgets/users/avatars/avatars.tsx

```diff
@@ -50,27 +57,47 @@ const displayNameGetter = makeDisplayNameGetter();
 
 function UserAvatar({
     userId,
-    overlayProps,
     ...props
 }: {
     userId: UserProfile['id'];
     overlayProps: Partial<ComponentProps<typeof SimpleTooltip>>;
 } & ComponentProps<typeof Avatar>) {
     const user = useSelector((state: GlobalState) => selectUser(state, userId)) as UserProfile | undefined;
-    const name = useSelector((state: GlobalState) => displayNameGetter(state, true)(user));
+
+    const overlay = React.createRef<MMOverlayTrigger>();
+
+    const getProfilePictureURL = (): string => {
+        if (userId) {
+            return Utils.imageURLForUser(userId);
+        }
+        return '';
+    };
 
     return (
-        <SimpleTooltip
-            id={`name-${userId}`}
-            content={name}
-            {...overlayProps}
+        <OverlayTrigger
+            ref={overlay}
+            trigger='click'
+            placement='right'
+            rootClose={true}
+            overlay={
+                <ProfilePopover
+                    className='user-profile-popover'
+                    userId={userId}
+                    src={getProfilePictureURL()}
+                />
+            }
         >
-            <Avatar
-                url={imageURLForUser(userId, user?.last_picture_update)}
-                tabIndex={0}
-                {...props}
-            />
-        </SimpleTooltip>
+            <button
+                className={'status-wrapper style--none'}
+                tabIndex={-1}
+            >
+                <Avatar
+                    url={imageURLForUser(userId, user?.last_picture_update)}
+                    tabIndex={0}
+                    size={props.size}
```

Comment:
I wasn't passing `{...props}` because it was complaining due to unexpected `%s` element in props. Actually overlayProps was creating the issue. Since we removed it above, there is no problem anymore. 

In the `Avatar` element, I didn't see `style` prop and in the old version there was no style prop passed, so I assume it is fine